### PR TITLE
Add the files if update files fails

### PR DESCRIPTION
**Pull request by bmalkowski**
In particular, I want to catch https://github.com/crowdin/crowdin-api/blob/master/lib/crowdin-api/errors.rb (which update_files raises) and if the error code is 8 (https://support.crowdin.com/api/error-codes/), try to add the file.

I'm not sure if I need to somehow catch that Crowdin Error class specifically or not.

### CODE REVIEW

File: fastlane/actions/crowdin_export.rb [hunk 1]
```diff
@@ -42,7 +42,13 @@ def self.run(config)
 
         UI.message("Exporting to Crowdin")
         crowdin = Crowdin::API.new(api_key: params[:api_key], project_id: params[:project_id])
-        crowdin.update_file(params[:files])
+        begin
+          crowdin.update_file(params[:files])
+        rescue => e
+          if e.error_code == 8
```
Comment (awesome) [inline fastlane/actions/crowdin_export.rb]: `rescue => e` catches every raised error, where `e` is the error, the redefined core `StandardError` here https://github.com/crowdin/crowdin-api/blob/master/lib/crowdin-api/errors.rb#L5 wouldn't necessarily be `e`

If you want to catch a specific error, use `rescue StandardError => e` where `e` will be the redefined `StandardError`

Comment (awesome) [reply fastlane/actions/crowdin_export.rb]: If an error is bubbling in the stack `rescue => e` will catch and not have the appropriate context, unless of course this logic was intentional…

Comment (awesome) [reply fastlane/actions/crowdin_export.rb]: I misspoke, crowdin subclassed `StandardError` as `Crowdin::API::Errors::Error`, namespace scoped with `Crowdin::API::Errors` --not redefined. So if you wanted to catch it by name: 

`rescue Crowdin::API::Errors::Error => e`

Comment (awesome) [reply fastlane/actions/crowdin_export.rb]: Just to specify, by "bubbling in the stack", I meant the calls triggered by `crowdin.update_file(params[:files])`; so basically all calls between `begin` and `rescue`

**Comment by bmalkowski** [comment]
I'll roll with using the `Crowdin::API::Errors::Error => e`. I'd prefer to be a bit more precise of the error. I kinda figured the other code would work but I think capturing the exact Crowdin Error would be better.

Thanks for taking a look!

**Pull request merged**